src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/Edit.cshtml

```diff
@@ -156,8 +156,8 @@
         <button class="btn btn-danger delete" type="submit" name="submit.Delete" value="Delete" data-url-af="RemoveUrl">@T["Delete"]</button>
     </div>
 </form>
-
-<script at="Foot">
+<script asp-name="Sortable"></script>
```

Comment:
This shouldn't be needed, since `depends-on` is like declaring a dependency in a resource manifest.

Reply:
I don't think so. I tried something similar in the pasts and without the script it won't work unless it's already being included by other scripts in the same request.  I think depends on only ensures the render order but not dependency. Unless something changed, this is the behavior I witnessed in the past.

Reply:
Hmm, in the manifest one uses `ResourceDefinition.SetDependencies`, and the tag helpers use `RequireSettings.SetDependencies`. Both are expanded in the same way in `ResourceManager.ExpandDependenciesImplementation`. But I also tried and adding a previously not included dependency is indeed not added. So, these inclusions are needed indeed.

Reply:
Re-added them,

Reply:
@Piedone  are you saying we should indeed add `<script asp-name="Sortable"></script>`? Note I removed it because in other places we do not explicitly add it. Here is one example

https://github.com/OrchardCMS/OrchardCore/blob/0e1d7b81a193f877440b7963e967c8788159566d/src/OrchardCore.Modules/OrchardCore.Deployment/Views/DeploymentPlan/Display.cshtml#L9

Reply:
Yes. I don't think this is good so, or that this was the intent, but currently it works like this.

Reply:
@Piedone if you can, please open an issue to track this one. We should find out why `depends-on` does not auto inject the resource if one is missing. I am guessing placement (which insert order do we place it) would be the reason. Even if placement is the reason, I don't think it is a good reason since we should be able to just place it immediately before the requiring script. If there is no good reason, `depends-on` should be enough to auto inject the dependencies.

Reply:
Opened it: https://github.com/OrchardCMS/OrchardCore/issues/16540